docs/context.rst

```diff
@@ -16,12 +18,16 @@ Raven defaults to the ``SingletonContextManager`` on Android, which maintains a
 context instance for all threads for the lifetime of the application.
 
 As of version ``8.0.0`` to override the ``ContextManager`` you will need to override
-the ``getContextManager`` method in the ``DefaultRavenFactory``. A simpler API will
+the ``getContextManager`` method in the ``DefaultRavenFactory``. A simpler API will likely
 be provided in the future.
 
 Using Breadcrumbs
 -----------------
 
+Breadcrumbs can be used to describe what was happening in an application leading
```

Comment:
Yeah I think I prefer "actions that occurred" since `events` is pretty specific in the docs.